base.go

```diff
@@ -11,9 +11,9 @@ import (
 
 // Result of RPC
 type Result struct {
-	Jsonrpc string      `json:"jsonrpc,omitempty"`
-	Result  interface{} `json:"result,omitempty"`
-	ID      string      `json:"id,omitempty"`
+	Jsonrpc string          `json:"jsonrpc,omitempty"`
+	Result  json.RawMessage `json:"result,omitempty"`
```

Comment:
@cubismic 
The purpose of the json.RawMessage is delaying JSON decoding.
See https://golang.org/pkg/encoding/json/#RawMessage